docs/running-on-yarn.md

```diff
@@ -54,8 +54,8 @@ In `yarn-cluster` mode, the driver runs on a different machine than the client,
 
 # Preparations
 
-Running Spark-on-YARN requires a binary distribution of Spark which is built with YARN support.
-Binary distributions can be downloaded from the Spark project website. 
+Running Spark on YARN requires a binary distribution of Spark which is built with YARN support.
```

Comment:
I don't see the value in the other changes; these two lines seem OK but this is pretty trivial. Would you consider taking one big review of the docs, or at least a logical set of docs, and identify meaningful improvements, and make a JIRA covering small improvements to a lot of docs? I don't think it's worth many tiny PRs for mostly non-functional changes.